src/main/java/com/iota/iri/cache/impl/CacheImpl.java

```diff
@@ -49,11 +49,12 @@ public V get(K key) {
             return null;
         }
         V value = strongStore.get(key);
-        if (value == null && weakStore.containsKey(key)) {
-            put(key, weakStore.get(key));
-            value = strongStore.get(key);
-            weakStore.remove(key);
-        }
+
+        // if (value == null && weakStore.containsKey(key)) {
+        // put(key, weakStore.get(key));
+        // value = strongStore.get(key);
+        // weakStore.remove(key);
+        // }
```

Comment:
Something here is missing, either we want to make values that we get more sticky...
Or we don't...

Anyhows, we have to check weakStore on cache get no?